src/main.py

```diff
@@ -101,6 +154,39 @@ async def get_handler(req):
 
 @app.route('/lists', methods=['GET'])
 async def get_handler(req):
+    """
+    Get all lists that are active for a given city.
+
```

Comment:
I'm being *really* pedantic here, but there's a double newline here while the other docstrings only have one